src/main/java/com/google/sps/servlets/NewsServlet.java

```diff
@@ -126,6 +126,7 @@ public String getHTML(HttpURLConnection connection) throws IOException {
       lastSubstring = substrings[i - 1];
       thisSubstring = substrings[i];
       title = thisSubstring.substring(thisSubstring.indexOf(">") + 1, thisSubstring.indexOf("<"));
+      title = title.replace("&#39;", "'");
```

Comment:
[Optional] Might be worth adding a note discussing why this has to handled specially.